app/goprotoc/goprotoc.go

```diff
@@ -51,12 +51,15 @@ func run(args []string, stdin io.Reader, stdout io.Writer, stderr io.Writer) err
 	}
 
 	if len(opts.inputDescriptors) > 0 && len(opts.includePaths) > 0 {
+		//lint:ignore ST1005 command return value
```

Comment:
There are enough of these that perhaps a `lint:file-ignore` at the top would be better.

Reply:
Done